netstandard/tools/targets/NETStandard.Library.targets

```diff
@@ -18,6 +18,10 @@
       <PropertyGroup>
         <HandlePackageFileConflictsAfter>ResolvePackageDependenciesForBuild</HandlePackageFileConflictsAfter>
         <HandlePublishFileConflictsAfter>RunResolvePublishAssemblies</HandlePublishFileConflictsAfter>
+        <RemoveDepsFileConflictsAfter>GenerateBuildDependencyFile</RemoveDepsFileConflictsAfter>
```

Comment:
There is also a `GeneratePublishDependencyFile` target because the .deps.json file can and will be different between `dotnet build` output and `dotnet publish`.  So you need to remove conflicts from the publish deps.json file as well.